<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultiplePing</class>
 <widget class="QWidget" name="MultiplePing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1059</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MultipleProcessesPing</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButtonPingBaidu">
     <property name="toolTip">
      <string>ping www.baidu.com</string>
     </property>
     <property name="text">
      <string>Ping Baidu</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QTextBrowser" name="textBrowserJD"/>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="pushButtonPingSanchor">
     <property name="toolTip">
      <string>ping www.sanchor.xyz</string>
     </property>
     <property name="text">
      <string>Ping Sanchor</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTextBrowser" name="textBrowserBaidu"/>
   </item>
   <item row="3" column="2">
    <widget class="QTextBrowser" name="textBrowserSanchor"/>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButtonPingBing">
     <property name="toolTip">
      <string>ping cn.bing.com</string>
     </property>
     <property name="text">
      <string>Ping Bing</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButtonPingJD">
     <property name="toolTip">
      <string>ping www.jd.com</string>
     </property>
     <property name="text">
      <string>Ping JD</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QTextBrowser" name="textBrowserBing"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
